tests/test-cases/Group1-Docker-Commands/1-04-Docker-Create.robot

```diff
@@ -34,6 +34,38 @@ Simple creates
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
 
+Simple Creates Verifying Folder Path
+    ${container1}=  Evaluate  'inventory1' + str(random.randint(1000,9999))  modules=random
+    ${container2}=  Evaluate  'inventory2' + str(random.randint(1000,9999))  modules=random
+    ${container3}=  Evaluate  'inventory3' + str(random.randint(1000,9999))  modules=random
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create --name ${container1} ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    Check VM Folder Path  ${container1}
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create --name ${container2} ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    Check VM Folder Path  ${container2}
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create --name ${container3} ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    Check VM Folder Path  ${container3}
+
+    # cleanup created containers
```

Comment:
I would suggest calling all cleanup functions using `TEARDOWN` (either test or suite level based on need), so to make sure they are run in any conditions instead of manually we doing it which is error prone.